Healthy/Classes/Extensions/UILabel+Style.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UILabel+Style.swift
+//  Healthy
+//
+//  Created by Abdallah on 14/05/2023.
+//
+
+import UIKit
+
+extension UILabel{
+    
+    // MARK:  Style tupe
+    
+    enum LabelStyle{
+        case title
+        case subtitle
+        case description
+        case textFieldTitleLabel
+    }
+    
+    // MARK:  apply style
+    
+    func applyLabelStyle(style: LabelStyle){
```

Comment:
```suggestion
    func applyStyle(_ style: LabelStyle){
```
As per @Ahattalla comment